### PR TITLE
Rewording abstract text again

### PR DESCRIPTION
**Pull request by martinthomson**
I had some time on the train; I'm not sure that this is better, but I didn't like the use of "server-sent events"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,13 +97,13 @@
...
-        The <cite>Push API</cite> provides <a title="webapp">webapps</a> with scripted access to
-        server-sent messages, for simplicity referred to here as <a title="push message">push
-        messages</a>, as delivered by <a title="push service">push services</a>. A <a>push
-        service</a> allows an <a>application server</a> to send messages to a <a>webapp</a>,
-        regardless of whether the <a>webapp</a> is currently active on the <a>user agent</a>. The
-        <a>push message</a> will be delivered to a <a>Service Worker</a>, which could then store
-        the message's data or display a notification to the user.
+        The <cite>Push API</cite> enables sending of a <a>push message</a> to a <a>webapp</a> via
+        a <a>push service</a>.  An <a>application server</a> can send a <a>push message</a> at any
+        time, even when the <a>webapp</a> is active or the <a>user agent</a> is offline.
```
Comment (mvano) [inline index.html]: Did you mean 'inactive'?

File: index.html [hunk 2]
```diff
@@ -97,13 +97,13 @@
...
+        The <a>push service</a> ensures reliable and efficient delivery to the <a>user
+        agent</a>.  <a title="push message">Push messages</a> are delivered to a <a>Service
```
Comment (mvano) [inline index.html]: Double spaces after periods will be stripped by html tidy (did you run it?). Also, they're not rendered.

**Comment by martinthomson** [comment]
Updated.

File: index.html [hunk 3]
```diff
@@ -97,13 +97,13 @@
...
-        The <cite>Push API</cite> provides <a title="webapp">webapps</a> with scripted access to
-        server-sent messages, for simplicity referred to here as <a title="push message">push
-        messages</a>, as delivered by <a title="push service">push services</a>. A <a>push
-        service</a> allows an <a>application server</a> to send messages to a <a>webapp</a>,
-        regardless of whether the <a>webapp</a> is currently active on the <a>user agent</a>. The
-        <a>push message</a> will be delivered to a <a>Service Worker</a>, which could then store
-        the message's data or display a notification to the user.
+        The <cite>Push API</cite> enables sending of a <a>push message</a> to a <a>webapp</a> via a
+        <a>push service</a>. An <a>application server</a> can send a <a>push message</a> at any
+        time, even when the <a>webapp</a> is active or the <a>user agent</a> is inactive. The
```
Comment (mvano) [inline index.html]: This is the more important question: "even when the webapp is active" sounds strange, certainly things usually work when they are active, it requires no special mention. Did you mean 'inactive'?

**Comment by martinthomson** [comment]
Yeah, that was a stuff-up on my part.  The point is that either can be inactive.

**Comment by mvano** [comment]
Looks good to me. Feel free to commit when you're ready.

**Pull request merged**